### PR TITLE
[lab][3] done

### PR DESCRIPTION
**Pull request by vityaman**

### CODE REVIEW

File: src/ucheb/1.sql [hunk 1]
```diff
@@ -0,0 +1,7 @@
+SELECT Н_ТИПЫ_ВЕДОМОСТЕЙ.ИД, Н_ВЕДОМОСТИ.ЧЛВК_ИД
+FROM Н_ТИПЫ_ВЕДОМОСТЕЙ
+RIGHT JOIN Н_ВЕДОМОСТИ
+ON Н_ТИПЫ_ВЕДОМОСТЕЙ.ИД = Н_ВЕДОМОСТИ.ВЕД_ИД
+WHERE Н_ТИПЫ_ВЕДОМОСТЕЙ.НАИМЕНОВАНИЕ = 'Перезачет'
+  AND Н_ВЕДОМОСТИ.ДАТА > '2022-06-08'
+  AND Н_ВЕДОМОСТИ.ДАТА = '2010-06-18';
```
Comment (akorton) [inline src/ucheb/1.sql]: Here and later where u have date comparison - though it seems to be working here, generally it is not a great idea to compare dates as strings.

Comment (vityaman) [reply src/ucheb/1.sql]: Added ::date suffix to interpret these strings as dates

**Comment by vityaman** [comment]
@akorton OK?

**Comment by akorton** [comment]
dont know how to OK the pr here :)

**Pull request merged**